content/news/09-02-21-the-open-revolution-talk

```diff
@@ -0,0 +1,20 @@
+title: "The Open Revolution: Making a radically fairer and free future"
+created: 2021-02-09
+slug: the-open-revolution-making-a-radically-fairer-and-free-future
+authors: [rufus-pollock]
+link: "https://www.youtube.com/watch?v=Em1l0QcR214"
+---
+
+## The Open Research Week - February 2021
+
+Rufus Pollock presented his talk *The Open Revolution: Making a radically fairer and free future* as part of The Open Research Week 2021. This event was hosted as a collaboration between the University of Liverpool, Liverpool John Mores University and Edge Hill University in February 2021.
+
+As the opening keynote, Rufus presented to the audience an examination of the true difference between the physical world and digital world, where our blocks often exist, and the need for us to move towards an open, as opposed to a closed, system.
+
+Discussing the key difference between costless copying when coupled with exclusive rights vs openness, Rufus presented the benefits and questions this raises, further emphasising the need for us to move away from the current monopoly model towards a renumeration rights model.
+
+The fast paced nature of the digital world can feel overwhelming which often means we only engage with it when we feel it is needed. However, understanding the underlying principles of the digital world is important. These principles do not change. Once these have been understood as underpinning everything within the digital world we can move forward with more useful discussions and actions to ensure that digital benefits all of us.
+
+You can watch the keynote below.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Em1l0QcR214" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
You should be able to just drop the youtube link direct and it just work (thanks to Flowershow framework we are using for the site) ... e.g.

```
https://www.youtube.com/watch?v=Em1l0QcR214
```